### ISSUE TITLE
Gutenberg: Video block should not be available on lower plans

### ISSUE THREAD

**Issue by jjchrisdiehl** [Automattic/wp-calypso#29254]
#### Steps to reproduce
1. Any WordPress.com Personal or lower site
2. /video add a video block
3. Upload a video
4. Warning messages that the video type is not supported

#### What I expected
Since we do not allow video hosting on Personal or lower, I expected it to not work.

#### What happened instead
The warning message does deny uploading a video, but the language is confusing. This ought to tell the user how to upgrade to take advantage of Videopress. 

Additionally, we have the Video Block with full video support, then we have the VideoPress block which simply lets you add a URL. It would be preferable if we chose one and had all the features included.

#### Browser / OS version
Chrome, Windows 10

#### Screenshot / Video

![image](https://user-images.githubusercontent.com/16580129/49668791-3a192980-fa2d-11e8-8902-68f31326fee4.png)

#### Context / Source
#paintpoint #manual-testing

**Comment by beaulebens** [comment]
I agree exactly with your suggestions there @MichaelArestad. We should definitely have a nudge towards upgrade so they can upload directly, but if we can fall back to at least allow them to add a link for any existing/external video, that would make a lot of sense.

**Comment by gwwar** [comment]
cc @mmtr if this happens to be closed by work in https://github.com/Automattic/wp-calypso/issues/29328
